includes/class-wc-gateway-komoju-ipn-handler.php

```diff
@@ -149,6 +149,13 @@ public function valid_response($webhookEvent)
      */
     public function validate_hmac($requestBody)
     {
+        $isDevEnv = getenv('WORDPRESS_DEV_ENV');
+        if ($isDevEnv) {
+            WC_Gateway_Komoju::log('Skipping IPN check in dev env');
+
+            return true;
+        }
```

Comment:
I think this hmac should work fine in dev :thinking: but I get that it can be a pain in the ass. The only issue I have with this check is that it makes it impossible to ever know if the hmac is working (without modifying the code to remove this check). 

Here's a suggestion: let's continue checking the hmac, even in dev. When it doesn't match up in dev, we can log "hmac didn't match" and then _return true anyway_. This means we're not losing the entire code path. What do you think? (basically I want to move this check down to line 168).

Reply:
Refactored based on your suggestions. We still check the hmac even in dev mode covering as much codebase as possible 👍🏻 
tests are still green so im going to merge this PR. thank you for review!